pkg/http/api.go

```diff
@@ -979,3 +992,44 @@ func getMetadataFromRequest(reqCtx *fasthttp.RequestCtx) map[string]string {
 
 	return metadata
 }
+
+func (a *api) onPostTransaction(reqCtx *fasthttp.RequestCtx) {
```

Comment:
If possible, please use the descriptive name for method such as `onPostStateTransaction()`